CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/components/ButtonToolbar.java

```diff
@@ -94,7 +95,7 @@ public class ButtonToolbar {
                 .append("Add or remove plugins from your Favourites category.\n\n")
                 .append("The following plugins were selected:\n")
                 .append("%s")
-                .append("\n\nNote, Favourites will be updated the next time you start Constellation.")
+                .append("\n\nNote, Favourites will be updated the next time you start " + BrandingUtilities.APPLICATION_NAME)
```

Comment:
are you able to add `BrandingUtilities...` to a new append? Generally with String builders you want to avoid concatenation within appends (and instead use appends themselves)